### DISCUSSION TITLE
Memory limit exceed

### DISCUSSION THREAD

**Discussion by jlerebours** [apache/doris#16524] [A - General / Q&A]
I'm currently having a problem of memory exceeded (OOM) running new doris version 1.2.1

Context : 
- both backend and frontend running in a container (kind of containerized standalone version) on my pc.
- my pc has like 3GB memory available and swap if neccessary

I have tried changing configurations of backend like : 
- mem_limit set to 3GB for now
- max_sys_mem_available_low_water_mark_bytes set to 600000000 (under 600Mb)

What I can see : 
- First, I have a problem with memory not auto detected on my computer but I don't think it is really a problem, here is the log from backend :
  `2023-02-08T12:05:51.324511763Z CGroup Info: Process CGroup Info: memory.limit_in_bytes=Could not find subsystem memory in /proc/self/cgroup, cpu cfs limits: Could not find subsystem cpuacct in /proc/self/cgroup`
- Then the most problematic thing happening sometimes even for a simple SELECT on 10 elements of database or just even to open table in editor (idea) :
`Process has no memory available, cancel top memory usage query: query memory tracker <Query#Id=14952ec332684010-9c95c9ddda5b9e5a> consumption 502.62 MB, backend 172.21.0.3 process memory used 596.14 MB exceed limit 3.00 GB or sys mem available 1.52 GB less than low water mark 572.20 MB. Execute again after enough memory, details see be.INFO.`

I really can't understand this log as limit is not really exceeded (572Mb < 3Gb) and this happens even on the lightest request.
What am I doing wrong ?

Thanks a lot for your help 

PS : I had no problem with 1.1.3 version without auto detection of memory and all but I need datetimeV2 so ...

**Comment by LemonLiTree** [comment]
Need to go to be.WARNING to see the information about mem_tracker
